payments-core/src/main/java/com/stripe/android/view/CardBrandViewRebuild.kt

```diff
@@ -0,0 +1,249 @@
+package com.stripe.android.view
+
+import android.content.Context
+import android.os.Parcelable
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import android.widget.FrameLayout
+import android.widget.ImageView
+import android.widget.ListPopupWindow
+import android.widget.TextView
+import com.stripe.android.R
+import com.stripe.android.databinding.StripeCardBrandViewRebuildBinding
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.CardBrand.Unknown
+import com.stripe.android.model.PaymentMethodCreateParams
+import kotlinx.parcelize.Parcelize
+import kotlin.properties.Delegates
+
+
+internal class CardBrandViewRebuild @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : FrameLayout(context, attrs, defStyleAttr) {
+    private val viewBinding: StripeCardBrandViewRebuildBinding =
+        StripeCardBrandViewRebuildBinding.inflate(LayoutInflater.from(context), this)
+    private val iconView = viewBinding.icon
+    private val progressView = viewBinding.progress
+    private val chevron = viewBinding.chevron
+    private val listPopup = ListPopupWindow(context)
```

Comment:
Any particular reason for `ListPopupWindow` over a `PopupMenu`? The latter should be a more direct replacement of what we have now.

Reply:
PopupMenu doesn't accept an Array Adapter which I need to display multiple view types in the menu